src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -393,8 +393,8 @@ public void validateMobility(MonitorableJob.Status status) {
             status.update("MobilityData Analysis...", 11);
 
             // Wait for the file to be entirely copied into the directory.
-            // TODO: base this on the file being completely saved rather than a fixed amount of time.
-            Thread.sleep(5000);
+            // 5 seconds + ~1 second per 10mb
+            Thread.sleep(5000 + (this.fileSize / 10000));
```

Comment:
What unit is `this.fileSize`? According to the code, it is set with `[newGtfs]File.length()` which is in bytes.

Reply:
@miles-grant-ibigroup Can you fix the comment so it corresponds to the code?

Reply:
Did I get the decimal math right in 0ebf2227078653ede1d54a77676b22bb4a89a726?

Reply:
I'm sorry, the initial comment was correct, juggling with the units was confusing (10,000,000 / 10,000 = 1000).